components/up.libsonnet

```diff
@@ -64,9 +63,9 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           '--duration=0',
           '--queries-file=/etc/up/queries.yaml',
           '--log.level=debug',
-        ],
+        ]
         +if std.objectHas(up.config, 'readEndpoint') then
-          ['--endpoint-read=' + up.config.readEndpoint] else [],
+          ['--endpoint-read=' + up.config.readEndpoint] else []
         +if std.objectHas(up.config, 'writeEndpoint') then
           ['--endpoint-write=' + up.config.writeEndpoint] else []
```

Comment:
There is a bug here and I am not sure why the `endpoint-write` doesn't get added [here](https://github.com/observatorium/configuration/pull/267/files#diff-e7c54c7f12b71107af51bc6dd182e74eR28)?